apps/rocketchat/template-rocketchat-mongodb.yaml

```diff
@@ -37,16 +38,18 @@ parameters:
     from: "[a-zA-Z0-9]{16}"
     required: true
 
+  # refer to the tools image
   - name: ROCKETCHAT_IMAGE_REGISTRY
     description: Location of the RocketChat Image
     displayName: RocketChat Image
-    value: docker.io/library/rocket.chat
+    value: "docker-registry.default.svc:5000/e5imao-tools/rocket.chat"
```

Comment:
I think we will need to make the project name a parameter. I believe there is a build in one for this?

Reply:
shall we then just keep an image stream in every single namespace instead of referring to tools, since there's no build?